aws/config.ts

```diff
@@ -53,36 +53,36 @@ export function mksClusterArgs(
       encryptionInTransit: {
         // Possible values are described here:
         // https://www.pulumi.com/docs/reference/pkg/aws/msk/cluster/#clusterencryptioninfoencryptionintransit
-        clientBroker: config.get<string>("msk-kafka-encryptionInfo-encryptionInTransit") || "TLS_PLAINTEXT",
+        clientBroker: config.get<string>("msk.kafka.encryptionInfo.encryptionInTransit") || "TLS_PLAINTEXT",
       },
     },
   };
 }
 
 export const akkaOperatorChartOpts: ChartOpts = {
   chart: "akka-operator",
-  version: config.get<string>("operator-version") || "1.1.19",
+  version: config.get<string>("akka.operator.version") || "1.1.21",
   fetchOpts: {
     repo: "https://lightbend.github.io/akka-operator-helm/",
   },
 };
 
-export const operatorNamespace = config.get<string>("operator-namespace") || LightbendNamespace;
+export const operatorNamespace = config.get<string>("akka.operator.namespace") || LightbendNamespace;
 
-export const installMetricsServer = getBooleanOrDefault("install-metrics-server", true);
-export const akkaOperatorServiceAccount = config.get<string>("akka-operator-service-account") || "sa";
-export const deployKafkaCluster = getBooleanOrDefault("deploy-kafka-cluster", true);
-export const deployJdbcDatabase = getBooleanOrDefault("deploy-jdbc-database", true);
+export const installMetricsServer = getBooleanOrDefault("kubernetes.metrics-server.install", true);
+export const deployKafkaCluster = getBooleanOrDefault("mks.createCluster", true);
+export const deployJdbcDatabase = getBooleanOrDefault("rds.createCluster", true);
 
-export const installAwsOTelCollector = getBooleanOrDefault("install-aws-otel-collector", false);
-export const awsOTelCollectorNamespace =
-  config.get<string>("aws-otel-collector-namespace") || AwsOTelCollectorNamespace;
+export const installTelemetryServices = getBooleanOrDefault("akka.operator.installTelemetryBackends", true);
+
+export const installAwsOTelCollector = getBooleanOrDefault("otel.collector.install", false);
+export const awsOTelCollectorNamespace = config.get<string>("otel.collector.namespace") || AwsOTelCollectorNamespace;
 // enables debug loglevel for the AWS OTel collector
-export const awsOTelCollectorDebug = getBooleanOrDefault("aws-otel-collector-debug", false);
-export const awsXRayRegion = config.get<string>("aws-xray-region") || new pulumi.Config("aws").get<string>("region");
-const awsXRayAccessKeyIDName = "aws-xray-access-key-id";
+export const awsOTelCollectorDebug = getBooleanOrDefault("otel.collector.debug", false);
```

Comment:
I've renamed corresponding settings in the platform docs: https://github.com/akka/akka-platform-guide/pull/741/commits/771a373df0ff15243dccf21b2bf9f229b621b420